src/app/pages/RoflAppDetailsPage/InstancesCard.tsx

```diff
@@ -28,13 +28,22 @@ const InstancesView: FC<RoflAppDetailsContext> = ({ scope, id }) => {
   })
   const { isFetched, isLoading, data } = instancesQuery
   const instances = data?.data.instances
+  const {
+    isFetched: isEpochFetched,
+    isLoading: isEpochLoading,
+    data: epochData,
+  } = useGetConsensusEpochs(scope.network, { limit: 1 })
+  const currentEpoch = epochData?.data.epochs[0].id
 
   return (
     <>
-      {isFetched && !instances?.length && <CardEmptyState label={t('rofl.emptyInstancesList')} />}
+      {isFetched && isEpochFetched && !instances?.length && !currentEpoch && (
```

Comment:
Wouldn't this show both the `CardEmptyState` & `InstancesList`. Or table doesn't render when you supply empty rows?